backend/src/nodes/utils/architecture/SwiftSRGAN.py

```diff
@@ -157,49 +158,3 @@ def forward(self, x):
         x = self.convblock(x) + initial
         x = self.upsampler(x)
         return (torch.tanh(self.final_conv(x)) + 1) / 2
-
-
-class Discriminator(nn.Module):
```

Comment:
Was this code unused or something?

Reply:
Discriminators are used in training. There's no reason for this to be here and it's just something I noticed when adding the subtype to this file